.github/workflows/pypi-release.yml

```diff
@@ -17,49 +17,17 @@ jobs:
         python-version: 3.7
 
     - name: Install dependencies
-      run: python -m pip install --user --upgrade setuptools wheel
+      run: python -m pip install --user --upgrade setuptools wheel build
     - name: Build source
-      env:
-        KORNIA_RELEASE: 1
       run: |
-        python setup.py sdist bdist_wheel
+        python -m build --sdist --wheel --outdir dist/
```

Comment:
>  By default, a source distribution (sdist) is built from {srcdir}
>     and a binary distribution (wheel) is built from the sdist.
>     This is recommended as it will ensure the sdist can be used
>     to build wheels.

```suggestion
        python -m build --outdir dist/
```